General/Operations/acknowledge.py

```diff
@@ -0,0 +1,15 @@
+# mission_ack(47) implementation
```

Comment:
Pretty minor, but in our other files we document #PROMISES and #REQUIRES under each MAVLink function. This one is pretty straightforward so I'm not sure how necessary it is here? Might be worth adding for consistency though.